maud/stan/functions.stan

```diff
@@ -822,7 +822,7 @@ vector maud_ae_system(vector conc,
                       array[,] int phosphorylation_ix_bounds,
                       array[] int phosphorylation_type,
                       array[] int phosphorylation_pme){
-  return conc - to_vector(ode_bdf_tol(dbalanced_dt, conc, 0, {1},
+  return conc - to_vector(ode_bdf_tol(dbalanced_dt, conc, 0, {1e-6},
```

Comment:
I think this line isn't meant to be part of this pr?

Reply:
Ye, sorry.